src/array.jl

```diff
@@ -729,6 +729,9 @@ function in(x::CategoricalValue, y::CategoricalArray{T, N, R}) where {T, N, R}
     end
 end
 
+Array(x::CategoricalArray{T}) where {T} = convert(Array{T}, x)
```

Comment:
Don't you need to import `Array`?

Reply:
Apparently not, the unit tests passes

Reply:
Yeah, but why? I've added an import since it sounds more correct. Better not rely on something we don't understand. :-)

Reply:
Fair enough :)